compiler-core/src/javascript/tests/snapshots/gleam_core__javascript__tests__sourcemaps__sourcemap_function_definition.snap

```diff
@@ -0,0 +1,50 @@
+---
+source: compiler-core/src/javascript/tests/sourcemaps.rs
+expression: sourcemap_viz
+---
+note: 
```

Comment:
This snapshot is rather verbose, to the extent of being hard to read. A more concise format would be very helpful

Reply:
I completely agree, but discussions with Gleam folks on discord didn't really lead to a precise conclusion. What exactly would you be happy with? 

Maybe each test could be "what is the Gleam location of this specific location in the JS file" - then the snapshot would contain the whole JS + Gleam code on top of each other, with the one position you're trying to assert? 

As an example, say you write a test case that asks "What is the position in the Gleam code of the character 1 line 1 in the generated JS file ?", the snapshot would look like this: 

```
  ┌─ original.gleam:1:1
  │
1 │ fn add_2(x) {
  │ ^ This code
   |    x + 2
   |  }
  │
  ┌─ generated.js:2:1
  │
2 │ function add_2(x) {
  │ ^ Gets mapped to this
   |     return x + 2;
   |  }
}
```